app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -40,6 +43,13 @@ const HEADER_FONT: &str = "DejaVuSans-Bold";
 
 
 
+// ===============
+// === Aliases ===
+// ===============
+
+type EntryId = list_view::entry::Id;
```

Comment:
why not importing `entry` and using `entry::Id` instead?

Reply:
Not everything from this entry module is related to component group. Perhaps in the future, when we will have more component-group- entry-specific stuff, we will have our own `entry` module there and `Id` reimported there.

Reply:
Add there short explanation about it, pls!